spec/features/home_page_spec.rb

```diff
@@ -92,4 +92,51 @@
       expect(find(@alert)).to have_content(@sign_in_msg)
     end
   end
+
+  describe 'contact us' do
+    before(:each) do
+      @name_textbox      = 'input#name'
+      @email_textbox     = 'input#email'
+      @select_menu       = 'select#subject'
+      @message_textarea  = 'textarea#message'
+      @message           = 'Hello, this is is cool.'
+      @send_button_text  = 'Send'
+      HydrusMailer.stub_chain(:contact_message, :deliver_now).and_return('a mailer')
```

Comment:
Deliver_now does not return anything, so there's no reason to stub its return value.

Reply:
ok, fixed